staff_features/accessions/accession_suppress.feature

```diff
@@ -6,17 +6,12 @@ Feature: Accession Suppress
   Scenario: Accession is suppressed
     Given the Accession is not suppressed
      When the user clicks on 'Suppress'
-      And the user clicks on 'Suppress' in the confirm popup
-     Then the Accession is suppressed
-      And the record should be hidden from non-authorized users
-      And the following info messages are displayed
-       | Accession suppressed                         |
-       | Accession is suppressed and cannot be edited |
+      And the user clicks on 'Suppress' in the modal
```

Comment:
I propose to use the same expression (in the 'confirm popup' instead of in the 'modal') as in the following 15 line